content/authors/ehis-edemakhiota/index.md

```diff
@@ -2,4 +2,4 @@
 title: Ehis Edemakhiota
 type: authors
 ---
```

Comment:
The author info is not according to the latest version. Please refer to the guidelines and modify it.